src/ui/hooks/recordings.ts

```diff
@@ -192,6 +192,35 @@ export async function getRecording(recordingId: RecordingId) {
   return result.data?.recording ? convertRecording(result.data.recording) : undefined;
 }
 
+export async function maybeAcceptInviteAndGetRecording(recordingId: RecordingId) {
+  const recording = await getRecording(recordingId);
+
+  if (!recording) {
```

Comment:
Should we check the error returned here? seems kind of odd to always assume there's a pending invite when the id may be invalid, or there may not be one, etc.

Reply:
Yeah, that makes sense 👍🏼  I realized that after writing this comment